fdbrpc/Locality.h

```diff
@@ -24,6 +24,11 @@
 
 #include "flow/flow.h"
 
+const StringRef ExcludeLocalityKeyDcIdPrefix = LiteralStringRef("dcid:");
```

Comment:
These should be `inline` to avoid any one definition rule* violations. Also, my personal preference is to us the `_sr` suffix string literal syntax (e.g. `"dcid:"_sr`), but I don't feel strongly enough to insist.

\* https://en.cppreference.com/w/cpp/language/definition